packages/babel-plugin/src/utils/ast.tsx

```diff
@@ -290,7 +290,12 @@ export const resolveBindingNode = (
       extensions: ['.js', '.jsx', '.ts', '.tsx'],
     });
     const moduleCode = fs.readFileSync(modulePath, 'utf-8');
-    const ast = parse(moduleCode, { sourceType: 'module', sourceFilename: modulePath });
+
+    const ast = meta.state.cache.load({
+      namespace: 'module-traversal',
+      cacheKey: `${moduleCode}//# modulePath=${modulePath}`,
```

Comment:
we can cache baseed on module path instead of using the code i think?

Reply:
🤔 What if module code changes between the builds? When we go for file caching?

Reply:
🤔 hmm. Let me try something.